src/components/organisms/ValidationDrawer/OpenPolicyAgent/ValidationOpenPolicyAgent.tsx

```diff
@@ -0,0 +1,43 @@
+import {useCallback} from 'react';
+import {useDispatch} from 'react-redux';
+
+import {reprocessAllResources, toggleAllRules} from '@redux/reducers/main';
+
+import {Button} from '@src/App.styled';
+
+import * as S from './ValidationOpenPolicyAgent.styled';
+import {ValidationOpenPolicyAgentHeading} from './ValidationOpenPolicyAgentHeading';
+import {ValidationOpenPolicyAgentTable} from './ValidationOpenPolicyAgentTable';
+
+type Props = {
+  onBack: () => void;
+};
+
+export function ValidationOpenPolicyAgent({onBack}: Props) {
```

Comment:
I guess it's my old muscle memory slipping in here. Any reason why we decided to go for React.FC? It's an additional hoop while a normal function suffices. I'll change it though in the spirit of a consistent codebase.

Reply:
Sure, let's leave it as just `Props` and we can use the longer names if we ever want to export them.

As far as React.FC goes, I like that it makes sure your returns are valid (helpful for example if you use `return undefined;` instead of the correct `return null;` when you don't want the component to render.